src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalThrowsCheck.java

```diff
@@ -41,21 +41,21 @@
  * </p>
  * <ul>
  * <li>
- * Property {@code illegalClassNames} - Specify throw class names to reject.
- * Type is {@code java.lang.String[]}.
- * Default value is {@code Error, RuntimeException, Throwable, java.lang.Error,
- * java.lang.RuntimeException, java.lang.Throwable}.
+ * Property {@code ignoreOverriddenMethods} - Allow to ignore checking overridden methods
```

Comment:
> just moving of content blocks in sync in all files.

If we are just moving and doing nothing else, then how did the `a` change from lowercase to capital?

Is there a way to prove that this is just re-ordering and no other changes?

Reply:
It is ordering and changing capital letters in message to simplify diff in macros.

There not that much places where lowercase is used. So we need to capitalize them to be consistent.

Reply:
Issue just says ordering.

Also removal from `MODULES_WITH_UNORDERED_PROPERTIES` is missing.

Reply:
https://github.com/checkstyle/checkstyle/blob/f61e24307e6b77cb2aa705226f492257fca03654/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java#L230

Reply:
I already forgot about this, let me see if this is same .

Reply:
fixed.